apps/nextjs-website/src/lib/strapi/homepage.ts

```diff
@@ -0,0 +1,44 @@
+import * as t from 'io-ts/lib';
+import * as qs from 'qs';
+import { fetchFromStrapi } from './fetchFromStrapi';
+
+const LinkHomepageCodec = t.strict({
+  text: t.string,
+  href: t.string,
+  target: t.union([
+    t.null,
+    t.literal('_self'),
+    t.literal('_blank'),
+    t.literal('_parent'),
+    t.literal('_top'),
+  ]),
+});
+
+export const StrapiHomepageCodec = t.strict({
+  data: t.strict({
+    id: t.number,
+    attributes: t.strict({
+      comingsoonDocumentation: t.type({
+        title: t.string,
+        links: t.array(LinkHomepageCodec),
```

Comment:
```suggestion
        links: t.array(LinkCodec),
```

Reply:
Same reason why I'd keep the `LinkHomepageCodec` name: https://github.com/pagopa/developer-portal/pull/615#discussion_r1485986971 and https://github.com/pagopa/developer-portal/pull/615#discussion_r1485988840.